include/envoy/server/worker.h

```diff
@@ -35,6 +35,20 @@ class Worker {
   virtual void stop() PURE;
 
   /**
+   * Remove a listener from the worker.
```

Comment:
Yeah I think we could do that eventually, but I would prefer to not do that in this PR. I have an ongoing branch where I'm implementing LDS which is spitting out all of these small PRs, so I'm already working on the drain, etc. case. I agree that we could potentially remove drained listeners faster, but the way I'm implementing LDS not sure it matters that much. I can put in a comment/TODO.